src/core/sys/dragonflybsd/dlfcn.d

```diff
@@ -0,0 +1,116 @@
+/**
+ * D header file for DragonFlyBSD
+ *
+ * Copyright: Copyright Martin Nowak 2012.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors: Martin Nowak,Diederik de Groot(port:DragonFlyBSD)
+ * Copied:  From core/sys/freebsd/sys
+ */
+module core.sys.dragonflybsd.dlfcn;
+
+public import core.sys.posix.dlfcn;
+
+version (DragonFlyBSD):
+extern (C):
+nothrow:
+@nogc:
```

Comment:
You can put all three attributes on the same line, ie `extern (C) nothrow @nogc:` as done for some other files below, probably better style.  Also, should most of these headers be `@system` too, @MartinNowak?

Reply:
Putting the attributes on one line -> No problem.
Adding @system attribute to all of these header files, i gave that a try and all the unittests are still passed without a problem. If that is an indication that it is ok, i can commit the change.

Reply:
Proposed
```diff
diff --git a/src/core/sys/dragonflybsd/dlfcn.d b/src/core/sys/dragonflybsd/dlfcn.d
index 9a416525..86effe89 100644
--- a/src/core/sys/dragonflybsd/dlfcn.d
+++ b/src/core/sys/dragonflybsd/dlfcn.d
@@ -11,9 +11,8 @@ module core.sys.dragonflybsd.dlfcn;
 public import core.sys.posix.dlfcn;
 
 version (DragonFlyBSD):
-extern (C):
-nothrow:
-@nogc:
+
+extern (C) nothrow @nogc @system:
 
 enum __BSD_VISIBLE = true;
 
diff --git a/src/core/sys/dragonflybsd/sys/_bitset.d b/src/core/sys/dragonflybsd/sys/_bitset.d
index 079115d7..24f9d9c1 100644
--- a/src/core/sys/dragonflybsd/sys/_bitset.d
+++ b/src/core/sys/dragonflybsd/sys/_bitset.d
@@ -7,7 +7,7 @@
 module core.sys.dragonflybsd.sys._bitset;
 
 version (DragonFlyBSD):
-extern (C) pure nothrow @nogc:
+extern (C) pure nothrow @nogc @system:
 
 import core.stdc.config : c_long;
 
diff --git a/src/core/sys/dragonflybsd/sys/elf_common.d b/src/core/sys/dragonflybsd/sys/elf_common.d
index cd94832d..4bbddf24 100644
--- a/src/core/sys/dragonflybsd/sys/elf_common.d
+++ b/src/core/sys/dragonflybsd/sys/elf_common.d
@@ -7,9 +7,8 @@
 module core.sys.dragonflybsd.sys.elf_common;
 
 version (DragonFlyBSD):
-extern (C):
-pure:
-nothrow:
+
+extern (C) pure nothrow @nogc @system:
 
 import core.stdc.stdint;
```
Etcetera for the rest of the files in this PR
Most of these could also be set to 'pure' i think, correct ?

Reply:
Not sure, haven't messed with `@system` much, will let someone who knows better answer.